pkg/ipamclaimswebhook/podmutator_test.go

```diff
@@ -105,11 +105,10 @@ var _ = Describe("KubeVirt IPAM launcher pod mutato machine", Serial, func() {
 
 		ipamClaimsManager := NewIPAMClaimsValet(mgr)
 
-		Expect(
-			ipamClaimsManager.Handle(context.Background(), podAdmissionRequest(config.inputPod)),
-		).To(
-			Equal(config.expectedAdmissionResponse),
-		)
+		result := ipamClaimsManager.Handle(context.Background(), podAdmissionRequest(config.inputPod))
+
+		Expect(result.AdmissionResponse).To(Equal(config.expectedAdmissionResponse.AdmissionResponse))
+		Expect(result.Patches).To(ConsistOf(config.expectedAdmissionResponse.Patches))
```

Comment:
We should do this per test not for all the test, since depending on the test it's important or not.

One way is to refactor the test so the `expectedPatches` is `gomega.Matcher` so we can do 

```golang
expectedPatch: ConsistOf(....)
```

Reply:
why? the order doesnt matter never on the patches field, is can always be ConsistOf,
consider please to keep it simple as it is now

Reply:
I understand, i think it is wrong, because we don't have it right now, and if we have then we can fix
atm we don't see we going to have such case, anyhow will change